lib/ea_restaurant_data_loader_web/error_handlers/error_handler.ex

```diff
@@ -0,0 +1,14 @@
+defmodule EaRestaurantDataLoaderWeb.ErrorHandlers.ErrorHandler do
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.InvalidCredentialsError
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.BadRequest
+
+  def custom_handle_errors(reason) do
+
+    case reason do

```

Comment:
You should return a `500 http error` if error is unknown

Reply:
Done.